packages/lodestar/src/eth1/impl/eth1Vote.ts

```diff
@@ -1,36 +1,14 @@
 import {IEth1Notifier} from "../interface";
 import {IBeaconConfig} from "@chainsafe/lodestar-config";
-import {BeaconState, Eth1Data, Number64} from "@chainsafe/lodestar-types";
-import {intSqrt} from "@chainsafe/lodestar-utils";
+import {BeaconState, Eth1Data} from "@chainsafe/lodestar-types";
 import {arrayIntersection, mostFrequent, sszEqualPredicate} from "../../util/objects";
-import {Block} from "ethers/providers";
+import {votingPeriodStartTime} from "./utils";
 
 export async function getEth1Vote(
-  this: IEth1Notifier, config: IBeaconConfig, state: BeaconState, previousEth1Distance: Number64
-): Promise<Eth1Data> {
-  const eth1Head = await this.getHead();
-  const allEth1Data = await getEth1DataRange(
-    this.getEth1Data,
-    eth1Head,
-    config.params.ETH1_FOLLOW_DISTANCE,
-    previousEth1Distance
-  );
-
-  const periodTail =
-        state.slot % config.params.SLOTS_PER_ETH1_VOTING_PERIOD
-        >=
-        intSqrt(config.params.SLOTS_PER_ETH1_VOTING_PERIOD);
-  let votesToConsider: Eth1Data[];
-  if(periodTail) {
-    votesToConsider = allEth1Data;
-  } else {
-    votesToConsider = await getEth1DataRange(
-      this.getEth1Data,
-      eth1Head,
-      config.params.ETH1_FOLLOW_DISTANCE,
-      2 * config.params.ETH1_FOLLOW_DISTANCE
-    );
-  }
+  this: IEth1Notifier, config: IBeaconConfig, state: BeaconState): Promise<Eth1Data> {
+  const periodStart = votingPeriodStartTime(config, state);
+  const promises: Promise<Eth1Data>[] = this.findBLocks(config, periodStart).map(block => this.getEth1Data(block));
```

Comment:
```suggestion
  const promises: Promise<Eth1Data>[] = this.findBlocks(config, periodStart).map(block => this.getEth1Data(block));
```

Reply:
rename method implementation also

Reply:
not sure if I understand it correctly. Do you mean to rename the `findBlocks` method?

Reply:
notice lowercase L

Reply:
I see it now lol